src/import/FSHTank.ts

```diff
@@ -22,20 +22,33 @@ export class FSHTank {
   }
 
   /**
-   * Finds the profile in the tank by name, if it exists
-   * @param {string} name - The name of the profile we're looking for
+   * Finds the profile in the tank by name, id, or alias, if it exists
+   * @param {string} key - The name or id of the profile we're looking for
    * @returns {Profile | undefined}
    */
-  public findProfileByName(name: string): Profile | undefined {
-    return this.getAllProfiles().find(profile => profile.name === name);
+  public findProfile(key: string): Profile | undefined {
+    return this.getAllProfiles().find(p => p.name === key || p.id === key);
   }
 
   /**
-   * Finds the extension in the tank by name, if it exists
-   * @param {string} name - The name of the extension we're looking for
-   * @returns {Extension | undefined}
+   * Finds the extension in the tank by name, id, or alias, if it exists
+   * @param {string} key - The name or id of the extension we're looking for
+   * @returns {[Extension, string]}
    */
-  public findExtensionByName(name: string): Extension | undefined {
-    return this.getAllExtensions().find(extension => extension.name === name);
+  public findExtension(key: string): Extension | undefined {
+    return this.getAllExtensions().find(p => p.name === key || p.id === key);
+  }
+
+  /**
+   * Finds the alias in the tank, if it exists
+   * @param {string} name - The name of the alias we're looking for
+   * @returns {string | undefined}
+   */
+  public resolveAlias(name: string): string | undefined {
```

Comment:
The logic of this function returns back the alias if it is not found, but the doc and the return type seems to indicate otherwise (implying it will return undefined if it is not found).  I feel like it may be important to know whether or not the alias actually resolved, which is difficult as implemented -- so my preference would be that it returns `undefined` when the alias cannot be resolved.  This would, of course, mean updating any code that relies on the current behavior.